proto/exocore/operator/v1/query.proto

```diff
@@ -294,4 +302,11 @@ service Query {
       get: "/exocore/operator/v1/opt/avs_list/{operator}"
     };
   }
+
+  // QueryOptInfo queries specified opted information.
+  rpc QueryOptInfo(QueryOptInfoRequest) returns (OptedInfo) {
+    option (google.api.http) = {
+      get: "/exocore/operator/v1/opt_info"
```

Comment:
Can this URL include the params like the one immediately above this one?

Reply:
Thanks, It has been fixed.